lib/foreman/renderer/doc_templates/basic_ruby_methods.rb

```diff
@@ -472,6 +472,16 @@ module Array
 a.fetch(-1)              #=> 44
 a.fetch(4, "cat")        #=> "cat"'
           end
+
+          apipie_method :rand, 'Returns random floating point numbers between 0 and 1' do
```

Comment:
I'd suggest to move it from `Array` class, since `rand` is not a method of an array object.
![ScreenShot-1618574958924](https://user-images.githubusercontent.com/32508194/115022258-71315c80-9ebd-11eb-8c01-84101d4325ce.png)

Reply:
Thanks for that! I totally miss that. Now I moved them to another place.